controllers/etcd_controller.go

```diff
@@ -978,6 +994,18 @@ func (r *EtcdReconciler) getMapFromEtcd(etcd *druidv1alpha1.Etcd) (map[string]in
 		sharedConfigValues["autoCompactionRetention"] = etcd.Spec.Common.AutoCompactionRetention
 	}
 
+	etcdConfigMountPath := "/var/etcd/config/"
+	initialCluster := fmt.Sprintf("%s=http://0.0.0.0:2380", etcd.Name)
+	if statefulsetReplicas > 1 {
+		etcdConfigMountPath = "/var/etcd/template/config/"
+		// form initial cluster
+		initialCluster = ""
+		for i := 0; i < statefulsetReplicas; i++ {
+			initialCluster = initialCluster + fmt.Sprintf("%s=http://%s-%d.%s:2380,", etcd.Name, etcd.Name, i, etcd.Name)
```

Comment:
```suggestion
			initialCluster = initialCluster + fmt.Sprintf("%s=http://%s-%d.%s:2380,", etcd.Name, etcd.Name, i, svc.Name)
```
Technically, the trailing name in the DNS should be service name (though it is the same as the `Etcd` resource name in this case).

Reply:
svc name is not available there.. So I improved the readability to let know that etcd name is being used as pod name and svc name